lib/ansible/modules/cloud/amazon/ec2_key.py

```diff
@@ -95,6 +95,25 @@
     state: absent
 '''
 
+RETURN = '''
+key.fingerprint:
```

Comment:
the 'key' key itself should be part of the docs, type complex and then you can show the 'subkeys' as part of it

Reply:
Complex return docs now are generated.